Formula/protobuf.rb

```diff
@@ -67,6 +67,10 @@ def install
         system python, *Language::Python.setup_install_args(prefix, python), "--cpp_implementation"
```

Comment:
```suggestion
        site_packages = prefix/Language::Python.site_packages(python)
        with_env(LDFLAGS: "-Wl,-rpath,#{rpath(source: site_packages)} #{ENV.ldflags}") do
          system python, *Language::Python.setup_install_args(prefix, python), "--cpp_implementation"
        end
```
To silence the warnings about files with missing rpaths.